x-pack/plugins/transform/public/app/sections/create_transform/components/step_create/step_create_form.tsx

```diff
@@ -191,35 +191,14 @@ export const StepCreateForm: FC<Props> = React.memo(
       const indexPatternName = transformConfig.dest.index;
 
       try {
-        const newIndexPattern = await indexPatterns.make();
-
-        Object.assign(newIndexPattern, {
-          id: '',
-          title: indexPatternName,
-          timeFieldName,
-        });
-        const id = await newIndexPattern.create();
-
-        await indexPatterns.clearCache();
-
-        // id returns false if there's a duplicate index pattern.
-        if (id === false) {
-          toastNotifications.addDanger(
-            i18n.translate('xpack.transform.stepCreateForm.duplicateIndexPatternErrorMessage', {
-              defaultMessage:
-                'An error occurred creating the Kibana index pattern {indexPatternName}: The index pattern already exists.',
-              values: { indexPatternName },
-            })
-          );
-          setLoading(false);
-          return;
-        }
-
-        // check if there's a default index pattern, if not,
-        // set the newly created one as the default index pattern.
-        if (!uiSettings.get('defaultIndex')) {
-          await uiSettings.set('defaultIndex', id);
-        }
+        const newIndexPattern = await indexPatterns.createAndSave(
```

Comment:
As above, this index pattern should have been created with `@timestamp.value_count` as the time field:

![image](https://user-images.githubusercontent.com/7405507/93774988-c7853800-fc19-11ea-8e8d-c639f59999bd.png)